src/lib/features/frontend-api/frontend-api-service.ts

```diff
@@ -170,56 +136,13 @@ export class FrontendApiService {
         if (!client) {
             client = this.createClientForFrontendApiToken(token);
             this.clients.set(token.secret, client);
-            this.config.eventBus.emit(PROXY_REPOSITORY_CREATED);
-        }
-
-        return client;
-    }
-
-    private async newClientForFrontendApiToken(
-        token: IApiUser,
-    ): Promise<Unleash> {
-        FrontendApiService.assertExpectedTokenType(token);
-
-        let newClient = this.newClients.get(token.secret);
-        if (!newClient) {
-            newClient = this.createNewClientForFrontendApiToken(token);
-            this.newClients.set(token.secret, newClient);
             this.config.eventBus.emit(FRONTEND_API_REPOSITORY_CREATED);
         }
 
-        return newClient;
-    }
-
-    private async createClientForFrontendApiToken(
-        token: IApiUser,
-    ): Promise<Unleash> {
-        const repository = new ProxyRepository(
-            this.config,
-            this.stores,
-            this.services,
-            token,
-        );
-        const client = new Unleash({
-            appName: 'proxy',
-            url: 'unused',
-            storageProvider: new InMemStorageProvider(),
-            disableMetrics: true,
-            repository,
-            disableAutoStart: true,
-            skipInstanceCountWarning: true,
-        });
-
-        client.on(UnleashEvents.Error, (error) => {
-            this.logger.error('We found an event error', error);
-        });
-
-        await client.start();
-
         return client;
     }
 
-    private async createNewClientForFrontendApiToken(
+    private async createClientForFrontendApiToken(
```

Comment:
Looking at lines 172 and 182 in this service, we did not do this for new clients during beta phase.
I renamed the `newClients` to `clients` and it auto-resolved, but I think it is correct to still have delete and destroy functions on the new clients.